lxd/storage_dir.go

```diff
@@ -334,12 +344,17 @@ func (s *storageDir) StoragePoolVolumeDelete() error {
 		return fmt.Errorf("no \"source\" property found for the storage pool")
 	}
 
+	_, err := s.StoragePoolUmount()
+	if err != nil {
+		return err
+	}
```

Comment:
Hmm, what? Why would you unmount the entire pool when attempting to delete a storage volume?

Reply:
Oh, I missed that one. I thought I had removed all umounts. Thanks!